src/extensions/default/InlineColorEditor/css/main.less

```diff
@@ -457,3 +457,5 @@
         }
     }
 }
+
+@sansFontFamily: Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
```

Comment:
those are three very WIndows specific fonts, all of which are from the Asian langauges... Where are the regular Sans serif fonts available in modern Windows, and where are the better OSX fonts? (and you probably want those at the front of the family, then, since Helvetica and Arial are considered websafe and will thus always get selected. Any system with the followup fonts has Arial, for instance)